### ISSUE TITLE
Remove Structure Pinpointer from Nuclear War

### ISSUE THREAD

**Issue by Mechswordsman** [tgstation/TerraGov-Marine-Corps#13654]
Essentially, the source for a lot of the 10 minute steamrolls on marine side, is for them to laser focus right to where the xeno base is.   Using hit and run tactics to try to pull apart marine balls is far less effective if they have something to aggressively rally towards. And 'delay and hiding structures' in ages past isn't really a issue with the mode either, since the marines can just do the disks to end the round.

TLDR, it is probably the simplest thing to improve round quality.

**Comment by KM-Catman** [comment]
If you're getting steamrolled by a simple silo push, you have other issues besides a crappy pinpointer

**Comment by Mechswordsman** [comment]
A simple silo push is one thing. Having the marines beeline directly to the silo every round is another. Less certainty in finding the silo, means less successful rushes and less rushes in general and would improve round quality.

**Comment by coldironwarrior** [comment]
Closed per #13656

**Comment by ivanmixo** [comment]
This is not something to put under the issues section
